.github/workflows/release.yml

```diff
@@ -68,3 +68,16 @@ jobs:
         env:
           GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  jira-release-version:
```

Comment:
nit: I see this process is skipped during the execution if it is a pre-release, just wondering if worth leaving a comment or explicit if statement in this job definition so this behaviour is clear without having to go into the implementation

Reply:
moved pre-release check to gha